### DISCUSSION TITLE
这是新功能，还是bug？

### DISCUSSION THREAD

**Discussion by Lacro2333** [asadahimeka/pixiv-viewer#8] [问题反馈] [answered]
最近突然要梯子才能使用了，刚开始用日本梯子还没问题，昨天连美国梯子打开软件，封面和标题都是日本画风的东西点进去就变成欧美画风的作品，然后点进搜索，下面的词条匹配一直别的，搜出来的东西也是，点开历史作者也是，合着给我变成了美服，我认为是我梯子的问题，重新连日本梯子再打开，内容正常了，但之前的bug还在，搜索搜不对，点开什么图都bug变成我刚开始点开的那张图，我把软件卸了重装都没变化，换一个版本也一样。

我又换梯子尝试，软件会根据国家不同改变你搜索的，点开的东西，现在软件完全用不了，搜什么都不对口，点什么都给我跳到同一张毫不相关的图

**Comment by asadahimeka** [comment]
@Lacro2333 
emmm 应该没有这个功能
我这边复现不出来这个问题
你用的是 Android 客户端还是 Windows 客户端
可以在网页版(`https://pxvek.cocomi.eu.org`)上试试有没有这个问题
客户端版本挺久没有更新了
网页版还是不行的话可以看看推荐(`https://pxvek.cocomi.eu.org/setting/recommend`)里的其他网站或软件
本人能力有限并且最近没有时间 不好意思

**Reply by Lacro2333** [reply]
我用的是Android客户端，最开始先发现网页端，然后又下载客户端的。
今天又尝试了一下，登录了就没问题了

**Reply by asadahimeka** [reply, marked as answer]
@Lacro2333 

可能是因为这个公告，所以最好用自己的账号登录后使用，而且要在官方网页端设置里面选择日区

https://www.pixiv.net/info.php?id=10837

<img width=300 src=https://github.com/asadahimeka/pixiv-viewer/assets/31837214/023fa943-9b11-4928-9cad-5663f2db69b0 >

**Comment by asadahimeka** [comment]
>  今天又尝试了一下，登录了就没问题了

最近发现图片不匹配或重复的问题是因为自建 API 的 CDN 缓存导致的，而登录以后不会使用自建 API，所以不会有这个问题😹
所以尽量登录后使用，另外 APP 最近更新了一版，可以[下载](https://github.com/asadahimeka/pixiv-viewer-apk/releases/download/1.0/+Pixiv-Viewer-1.17.5.apk)试试
